packages/react-common/test-manual/react-common-manual-test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { CovidMoratoriumBanner } from "../src";
+import { getHTMLElement } from "@justfixnyc/util";
+
+const container = getHTMLElement("div", "#root");
+const el = ReactDOM.render(
+  <div>
+    <h1>English Version (Default):</h1>
+    <CovidMoratoriumBanner />
+    <h1>Spanish Version:</h1>
+    <CovidMoratoriumBanner locale="es" />
```

Comment:
Since the package is called `react-common` and not `covid-moratorium-banner`, maybe the structure of the headings should be something like this:

```
Components (h1)
  COVID Moratorium Banner (h2)
    English version (h3)
    Spanish version (h3)
```

What do you think?  That way it'd be straightforward how to add more to this page if/when we add more components but it's up to you.

Reply:
Yes that makes sense to me!